src/table/pagination-table/components/TableWrapper.tsx

```diff
@@ -48,6 +51,10 @@ function TableWrapper(props: TableWrapperProps) {
     tableContainerRef.current?.scrollTo(0, 0);
   }, [pageInfo, totalRowCount]);
 
+  useEffect(() => {
+    setBorderHeight(paginationTableRef.current?.getBoundingClientRect?.()?.height);
+  }, [paginationTableRef, rootElement, rows.length]);
```

Comment:
you probably need more deps here. For example, the table height can change on when columnWidths changes... That makes it bit more tricky

Reply:
https://user-images.githubusercontent.com/4471489/225877878-00f80f7a-98f3-43f4-b092-50a4f16f7fbc.mov